src/OSPSuite.Core/Domain/Services/ParameterIdentifications/IResidualCalculator.cs

```diff
@@ -7,5 +8,6 @@ public interface IResidualCalculator
    {
       void Initialize(RemoveLLOQMode removeLLOQMode);
       ResidualsResult Calculate(IReadOnlyList<SimulationRunResults> simulationsResults, IReadOnlyList<OutputMapping> allOutputMappings);
+      ResidualsResult CalculateForSimulation(DataRepository simulationResultRepository, IReadOnlyList<OutputMapping> allOutputMappings);
```

Comment:
Probably this class should be moved to another namesepase?
Also call the method Calculate (overload?)

Reply:
Moving to another namespace done. About the renaming, do you think I should add a comment regarding where each overload is used (PI or Simulation), or better it is not necessary since it could actually theoretically be used also in other cases?